### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by sbkim052** [hche11/VGGSound#13]
Thank you for sharing a wonderful dataset.
Could you explain about the training settings, such as learning rate, weight decay, scheduler, etc.?
It is kind of hard to reproduce similar accuracy.
